action.yml

```diff
@@ -12,4 +16,4 @@ runs:
   using: "composite"
   steps:
     - shell: bash
-      run: ${{ github.action_path }}/run.sh ${{ inputs.directory }} ${{ inputs.release-branch-prefix }}
+      run: ${{ github.action_path }}/run.sh "${{ inputs.directory }}" "${{ inputs.release-branch-prefix }}" "${{ inputs.npm-build-command }}"
```

Comment:
```suggestion
      run: ${{ github.action_path }}/run.sh '${{ inputs.directory }}' '${{ inputs.release-branch-prefix }}' '${{ inputs.npm-build-command }}'
```

Reply:
Looks good! That should prevent any string interpolation.